chalice/deploy/deployer.py

```diff
@@ -421,11 +421,25 @@ def deploy(self, config, existing_resources, stage_name):
                                    deployed_values)
         self._deploy_event_sources(config, existing_resources, stage_name,
                                    deployed_values)
+        self._deploy_pure_lambda_functions(config, existing_resources,
+                                           stage_name, deployed_values)
         if existing_resources is not None:
             self._cleanup_unreferenced_functions(existing_resources,
                                                  deployed_values)
         return deployed_values
 
+    def _deploy_pure_lambda_functions(self, config, existing_resources,
```

Comment:
It will just override the first lambda function (it will trigger the function already exists check which will make it think it's an update instead of an initial create).

I can add validation for this.  It's a little more involved because you have to check uniqueness across all lambda functions we deploy (scheduled events, custom auth, pure lambda).